### ISSUE TITLE
Unreal 4.25 is supported?

### ISSUE THREAD

**Issue by qpaozixu** [zkarmakun/ARToolKit-5.3-for-Unreal-Engine#2]
Is this plugin support unreal 4.25?

**Comment by zkarmakun** [comment]
No, I am not supporting this project any longer, since Artoolkit is old and UE does have a better implementation with android and iOS
